metadata-io/src/main/java/com/linkedin/metadata/graph/GraphService.java

```diff
@@ -79,11 +80,12 @@ public interface GraphService {
    *   - RelatedEntity("DownstreamOf", "dataset three")
    */
   @Nonnull
-  RelatedEntitiesResult findRelatedEntities(@Nullable final String sourceType, @Nonnull final Filter sourceEntityFilter,
-      @Nullable final String destinationType, @Nonnull final Filter destinationEntityFilter,
+  RelatedEntitiesResult findRelatedEntities(@Nullable final List<String> sourceTypes, @Nonnull final Filter sourceEntityFilter,
```

Comment:
Minor thing : We don't really need to remove this API, we could just have the old API call the new API with the different arguments.

```
RelatedEntitiesResult findRelatedEntities(final String sourceType, Filter filter,...) {
   return findRelatedEntities(ImmutableList.of(sourceType, ...);
}

RelatedEntitiesResult findRelatedEntities(final List<String> sourceType, Filter filter,...) {

}
```

Reply:
This actually creates ambiguity if the user provides null for both fields (which is something we support), so I'll keep it as is for now.